src/styles/index.scss

```diff
@@ -1,3 +1,68 @@
-body {
+@import 'variables';
+
+html {
+  font-family: Arial, sans-serif;
+}
+
+* {
   margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+body {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  height: 100vh;
+}
+
+@each $day, $index in $days {
+  $index: $index + 6;
+  .calendar--start-day-#{$day} > :first-child {
+    margin-left: calc((#{$calendar-day-size} + #{$calendar-gap}) * #{$index});
+  }
+}
```

Comment:
The SCSS variable `$index` is being modified inside the `@each` loop which is not recommended. It's better to define a new variable inside the loop if you need to calculate a derived value. This ensures that the original `$index` value provided by the loop is not altered, which could lead to confusion or unexpected behavior.